File: ООП/Homework2/all_animals/stork.java
package all_animals;

public class stork extends birds {

    protected stork(int height, int weight, String eyes_color, int flight_altitude) {
        super(height, weight, eyes_color, flight_altitude);
    }

    public void fly() {
        String.format("%sВысота полёта: %d метров", super.flight_altitude);
    }

    public void voice() {
        System.out.println("\nЧик-чик-чик(?)");
    }

    public String toString(){
        return String.format("%s \nАист: %s", super.toString());
    }
    
}

File: ООП/Homework2/animals_PY/classes/birds/stork.py
from ..interfaces.flying import flying
from .bird import bird

class stork(bird, flying):
    def __init__(self, name, height, weight, coloreye, flying_height):
        super().__init__(name, height, weight, coloreye, flying_height)

    def voice(self):
       print(f"{super().voice()} чик-чик-чик")

    def fly(self):
        flying.fly(self)
    
    def __str__(self) -> str:
        return super().__str__()